### DISCUSSION TITLE
How to turn on the podreset in k3s

### DISCUSSION THREAD

**Discussion by FLM210** [k3s-io/k3s#8285] [Q&A] [answered]
I added two startup parameters to the K3s service configuration file  
`  --kube-apiserver-arg=enable-admission-plugins=NodeRestriction,PodPreset
--kube-apiserver-arg=runtime-config=settings.k8s.io/v1alpha1=true`
But the service reported an error
`  time="2023-08-31T18:26:37+08:00" level=fatal msg="apiserver exited: [enable-admission-plugins plugin \"PodPreset\" is unknown, unknown api groups settings.k8s.io]"`

_Originally posted by @FLM210 in https://github.com/k3s-io/k3s/issues/1604#issuecomment-1700778445_

**Comment by FLM210** [comment]
My k3s version is v1.24.9

**Comment by brandond** [comment]
https://github.com/k3s-io/k3s/issues/1604#issuecomment-1701538229

**Comment by FLM210** [comment, marked as answer]
PodPreset has been removed in version 1.20  https://github.com/kubernetes/kubernetes/pull/94090 😣

**Reply by brandond** [reply]
Yeah, guess I assumed you knew what you were doing and were working with an old version of k3s.
